e2e/benchmarks/benchmark_util.js

```diff
@@ -502,8 +503,9 @@ const TUNABLE_FLAG_VALUE_RANGE_MAP = {
 /**
  * Set environment flags for testing.
  *
- * This is a wrapper function of `tf.env().setFlags()` to constrain users to
- * only set tunable flags (the keys of `TUNABLE_FLAG_TYPE_MAP`).
+ * This will first set tunable flags (the keys of `TUNABLE_FLAG_TYPE_MAP`). Then
+ * set URL parameter flags. If there are overlap, URL parameter flags will
+ * override tunable flags.
```

Comment:
We also need to change the description here accordingly.

Reply:
Do you have any detailed suggestion here?

Reply:
LGTM